cortex-mixin/dashboards/alertmanager.libsonnet

```
@@ -83,5 +87,150 @@ local utils = import 'mixin-utils/utils.libsonnet';
+      .addPanel(
+        $.panel('Syncs/sec') +
+        $.queryPanel(
+          [
+            |||
+              sum(rate(cortex_alertmanager_sync_configs_total{%s}[$__rate_interval]))
+              -
+              sum(rate(cortex_alertmanager_sync_configs_failed_total{%s}[$__rate_interval]))
+            ||| % [$.jobMatcher('alertmanager'), $.jobMatcher('alertmanager')],
+            'sum(rate(cortex_alertmanager_sync_configs_failed_total{%s}[$__rate_interval]))' % $.jobMatcher('alertmanager'),
+          ],
+          ['success', 'failed']
+        )
+      )
+      .addPanel(
+        $.panel('Syncs/sec (By Reason)') +
+        $.queryPanel(
+          'sum by(reason) (rate(cortex_alertmanager_sync_configs_total{%s}[$__rate_interval]))' % $.jobMatcher('alertmanager'),
+          '{{reason}}'
+        )
+      )
+      .addPanel(
+        $.panel('Ring Check Errors/sec') +
+        $.queryPanel(
+          'sum (rate(cortex_alertmanager_ring_check_errors_total{%s}[$__rate_interval]))' % $.jobMatcher('alertmanager'),
+          'errors'
+        )
+      )
+    )
+    .addRow(
+      $.row('Sharding Initial State Sync')
+      .addPanel(
+        $.panel('Syncs/sec') +
+        $.queryPanel(
+          [
+            |||
+              sum(rate(cortex_alertmanager_state_initial_sync_total{%s}[$__rate_interval]))
+              -
+              sum(rate(cortex_alertmanager_state_initial_sync_completed_total{outcome="failed",%s}[$__rate_interval]))
+            ||| % [$.jobMatcher('alertmanager'), $.jobMatcher('alertmanager')],
+            'sum(rate(cortex_alertmanager_state_initial_sync_completed_total{outcome="failed",%s}[$__rate_interval]))' % $.jobMatcher('alertmanager'),
+          ],
+          ['success', 'failed']
+        )
+      )
+      .addPanel(
+        $.panel('Syncs/sec (By Outcome)') +
+        $.queryPanel(
+          'sum by(outcome) (rate(cortex_alertmanager_state_initial_sync_completed_total{%s}[$__rate_interval]))' % $.jobMatcher('alertmanager'),
+          '{{outcome}}'
+        )
+      )
+      .addPanel(
+        $.panel('Duration') +
+        utils.latencyRecordingRulePanel('cortex_alertmanager_state_initial_sync_duration_seconds', $.jobSelector('alertmanager'))
+      )
+    )
+    .addRow(
+      $.row('Sharding State Operations')
+      .addPanel(
+        $.panel('Replica Fetches/sec') +
+        $.queryPanel(
+          [
+            |||
+              sum(rate(cortex_alertmanager_state_fetch_replica_state_total{%s}[$__rate_interval]))
+              -
+              sum(rate(cortex_alertmanager_state_fetch_replica_state_failed_total{%s}[$__rate_interval]))
+            ||| % [$.jobMatcher('alertmanager'), $.jobMatcher('alertmanager')],
+            'sum(rate(cortex_alertmanager_state_fetch_replica_state_failed_total{%s}[$__rate_interval]))' % $.jobMatcher('alertmanager'),
+          ],
+          ['success', 'failed']
+        )
+      )
+      .addPanel(
+        $.panel('Replica Updates/sec') +
```

I would try to clarify a bit what's that.

```suggestion
        $.panel('Replicate state to other alertmanagers /sec') +
```